e2e/miniconstellation/BUILD.bazel

```
@@ -0,0 +1,33 @@
+load("//bazel/sh:def.bzl", "sh_template")
+
+filegroup(
+    name = "dependencies",
+    srcs = [
+        ".terraform.lock.hcl",
+        "cloud-init.yaml",
+        "main.tf",
+        "output.tf",
+        "test-remote.sh",
+    ],
+)
+
+sh_library(
+    name = "dependencies_lib",
+    srcs = [
+        ":dependencies",
+    ],
+)
+
+sh_template(
+    name = "miniconstellation_test_e2e_remote",
```

```suggestion
    name = "remote_test",
```
As this target is invoked via a path, I would argue it is okay to simplify this...